projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.html

```diff
@@ -5,47 +5,54 @@ <h3 class="cx-review-title d-none d-lg-block d-xl-block">
   </h3>
 
   <div class="cx-review-summary row">
-    <div
-      class="col-md-12 col-lg-6 col-xl-4"
-      *ngFor="let step of (steps$ | async).slice(0, -1)"
-    >
-      <div class="cx-review-summary-wrapper" [ngSwitch]="step.type[0]">
-        <ng-container *ngSwitchCase="checkoutStepType.PAYMENT_TYPE">
-          <ng-container *ngTemplateOutlet="poNumber"></ng-container>
-        </ng-container>
-        <ng-container *ngSwitchCase="checkoutStepType.PAYMENT_TYPE">
-          <ng-container *ngTemplateOutlet="paymentType"></ng-container>
-        </ng-container>
-        <ng-container *ngSwitchCase="checkoutStepType.SHIPPING_ADDRESS">
-          <ng-container *ngTemplateOutlet="costCenter"></ng-container>
+    <ng-container *ngIf="(steps$ | async).slice(0, -1) as steps">
+      <div class="col-md-12 col-lg-6 col-xl-6 cx-review-payment-col">
+        <ng-container *ngFor="let step of steps">
```

Comment:
just a food for thought, maybe we can separate the list into shipping and payment so we have shippingSomething$ and paymentSomething$ which will help divide the left and right side of the columns instead of looping through the whole thing again